Source/bmalloc/bmalloc/DebugHeap.cpp

```diff
@@ -106,6 +106,55 @@ void DebugHeap::dump()
     malloc_zone_print(m_zone, verbose);
 }
 
+#elif BOS(WINDOWS)
+
+// DebugHeap unimplemented on Windows
```

Comment:
Let's mention to the reason why (impedance mismatch between free and aligned free)